test/ipfs-test-lib.sh

```diff
@@ -109,3 +109,13 @@ test_seq() {
 		i=$(expr "$i" + 1)
 	done
 }
+
+b64decode() {
+    for flag in "-D" "-d"; do
+        if base64 "$flag" 2>/dev/null; then
+            return
+        fi
+    done
+    echo "no compatible base64 command found" >&2
+    return 1
```

Comment:
I am not sure how I feel about this little hack.  What if one of the base64 supports both options and they do two different things?

Maybe a simpler and more portable solution would be to write our own tool in go and use that, or install someone else's tool in `test/sharness/bin`.

Reply:
Meh, it's a test case. MacOS uses `-D` and everything else uses `-d`. We could install and use a different command but I'd rather not unless you *really* think this hack is going to be a problem.

Reply:
Could we at least switch based on the OS?  I will be okay with that.

It could be fine, but it could also create a hard to track down bug, several years out if by the very off chance `-D` does something other then decode in a future version.

Reply:
@kevina fair enough. I was trying to be smart but you're right, being smart is probably a bad idea in this case.